conf/modules/cv_colorfilter.xml

```diff
@@ -3,6 +3,7 @@
 <module name="ColorFilter" dir="computer_vision">
   <doc>
     <description>ColorFilter</description>
+    <configure name="COLORFILTER_CAMERA" value="VIDEO1|VIDEO2" description="Video device to use"/>
```

Comment:
Just to clarify, what is the difference between `define` and `configure`? Why does `configure` allow a default value and `define` does not?